src/js/components/DateInput/DateInput.js

```diff
@@ -105,17 +109,17 @@ const DateInput = forwardRef(
           disabled
             ? undefined
             : (nextValue) => {
-                let normalizedValue;
```

Comment:
I'm wondering why these changes are being added. Do you have the latest `prettier` installed?

Reply:
Thank you for the review.
Yes, I don't know why it has done that.
I have reverted those changes.